src/platforms/px4_defines.h

```diff
@@ -142,107 +167,79 @@ typedef param_t px4_param_t;
 /* FIXME - Used to satisfy build */
 #define getreg32(a)    (*(volatile uint32_t *)(a))
 
-#ifdef __PX4_QURT
-#define PX4_TICKS_PER_SEC 1000L
-#else
-__BEGIN_DECLS
-extern long PX4_TICKS_PER_SEC;
-__END_DECLS
-#endif
-
 #define USEC_PER_TICK (1000000UL/PX4_TICKS_PER_SEC)
```

Comment:
@mcharleb Thanks for reviewing this! A good question, but no. Macro's are expanded recursively when they are used, not when they are defined, so the order of defining them doesn't matter.

Ie,
```
#define FOO hello
#define BAR FOO world
BAR
```
expands to:
```
>g++ -P -E troep.cc
hello world
```

But also
```
#define BAR FOO world
#define FOO hello
BAR
```
gives
```
hello world
```
and, note that
```
#define FOO hello
#define BAR FOO world
#undef FOO
BAR
```
still expands to
```
FOO world
```